pkg/controller/apiserver/apiserver.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiserver
+
+import (
+	"github.com/vmware-tanzu/antrea/pkg/controller/apiserver/handlers/endpoint"
+	"github.com/vmware-tanzu/antrea/pkg/controller/networkpolicy"
+	genericapiserver "k8s.io/apiserver/pkg/server"
+)
+
+func InstallHandlers(eq networkpolicy.EndpointQueryReplier, s *genericapiserver.GenericAPIServer) {
+	s.Handler.NonGoRestfulMux.HandleFunc("/endpoint", endpoint.HandleFunc(eq))
```

Comment:
Could you move it to pkg/apiserver/apiserver.go where we install APIGroups?
For the required  EndpointQueryReplier, maybe inject it to `ExtraConfig` of APIServer like other dependencies.

Reply:
@antoninbas

Reply:
@tnqn would also like Jake to move handlers from `pkg/controller/apiserver/handlers` to ` pkg/apiserver/handlers`?

Reply:
I think so, given that we actually put all controller specific apiserver code there.